.thesun/publish-history.md

```
@@ -0,0 +1,70 @@
+# tessie MCP Publish History
+
+This file tracks where documentation has been published.
+DO NOT commit to public repositories.
+
+## Local
+- Path: /Users/timothy.schwarz/Scripts/mcp-servers/tessie-mcp-local
+- Forked from: https://github.com/keithah/tessie-mcp
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)